login.js

```diff
@@ -0,0 +1,32 @@
+const passwordElem = document.getElementById("selected-user-password");
+const loginElem = document.querySelector(".login-btn");
+const selectElem = document.getElementById("select");
+
+let registeredUsers = JSON.parse(localStorage.getItem("MB_USER_ACCOUNTS"));
+for (let i = 0; i < registeredUsers.length; i++) {
+  let optionElem = document.createElement("option");
+  let text = document.createTextNode(registeredUsers[i].accountName);
+  optionElem.appendChild(text);
+  optionElem.setAttribute("value", registeredUsers[i].accountNumber);
+  selectElem.insertBefore(optionElem, selectElem.lastChild);
+}
+
+// function for logging in
+function login() {
+  localStorage.setItem(
+    "MB_LOGGEDIN_USER_ACCOUNT_NUMBER",
+    JSON.stringify(selectElem.value)
+  );
+  if (localStorage.getItem("MB_LOGGEDIN_USER_ACCOUNT_NUMBER") !== null) {
+    location.href = "transaction.html";
+  }
+  //to check user object with the stored account number
+  let accountUserDetails = registeredUsers.find(
+    (acctNo) =>
+      acctNo.accountNumber ===
+      JSON.parse(localStorage.getItem("MB_LOGGEDIN_USER_ACCOUNT_NUMBER"))
+  );
+  console.log(accountUserDetails);
+}
```

Comment:
You need to include the logic for redirecting to the Transactions History page upon successful login. 

You can also check if the user exists in our database (although it's necessary in our case but that's a fundamental part of the login action).

Suggested:

```
if (!accountUserDetails) {
  alert('Account not found!');
  return;
}
window.location.href = "transactions.html";
```

Note that the Transactions History page should be named "transactions.html", not "transaction.html", because on it we can see many transactions, not just one. Use the same convention for other page names.